noir-projects/noir-contracts/contracts/ecdsa_account_contract/src/main.nr

```diff
@@ -40,6 +40,14 @@ contract EcdsaAccount {
     }
 
     #[aztec(private)]
+    #[aztec(noinitcheck)]
+    fn pay_init_fee(fee_payload: pub FeePayload) {
+        let actions = AccountActions::private(&mut context, ACCOUNT_ACTIONS_STORAGE_SLOT, is_valid_impl);
+        actions.pay_init_fee(fee_payload);
+    }
```

Comment:
The idea is to delete this method once we can figure out the hash mismatch, right?

Reply:
Yep, here's the PR #5623